frontend/test/metabase/scenarios/collections/collections.cy.spec.js

```diff
@@ -136,6 +137,61 @@ describe("scenarios > collection defaults", () => {
     });
   });
 
+  describe("render last edited by when names are null", () => {
+    beforeEach(() => {
+      restore();
+      cy.signInAsAdmin();
+    });
+
+    it("should render short value without tooltip", () => {
+      cy.intercept(
+        "GET",
+        "/api/collection/root/items?models=dashboard**",
+        req => {
+          req.on("response", res => {
+            res.send(
+              set(res.body, ["data", 0, "last-edit-info"], {
+                id: 1,
+                last_name: null,
+                first_name: null,
+                email: "admin@metabase.test",
+                timestamp: "2022-07-05T07:31:09.054-07:00",
+              }),
+            );
+          });
+        },
+      );
+      visitRootCollection();
+      cy.findByText("admin@metabase.test").trigger("mouseenter");
+      cy.findByRole("tooltip").should("not.exist");
+    });
+
+    it("should render long value with tooltip", () => {
+      cy.intercept(
+        "GET",
+        "/api/collection/root/items?models=dashboard**",
+        req => {
+          req.on("response", res => {
+            res.send(
+              set(res.body, ["data", 0, "last-edit-info"], {
+                id: 1,
+                last_name: null,
+                first_name: null,
+                email: "averyverylongemail@veryverylongdomain.com",
+                timestamp: "2022-07-05T07:31:09.054-07:00",
+              }),
+            );
+          });
+        },
+      );
+      visitRootCollection();
+      cy.findByText("averyverylongemail@veryverylongdomain.com").trigger(
+        "mouseenter",
+      );
+      cy.findByRole("tooltip").should("exist");
```

Comment:
Test that we show tooltip when the value is long and got ellipsised 
![image](https://user-images.githubusercontent.com/1937582/177359214-a4953d05-d318-4744-b96f-de0bfb8b615b.png)